ReactCommon/cxxreact/Value.h

```diff
@@ -13,6 +13,7 @@
 #include <JavaScriptCore/JSValueRef.h>
 
 #include <folly/dynamic.h>
+#include <jni/LocalString.h>
```

Comment:
@rigdern: to be clear, in order to land this we're going to need to be able to access the utf16 conversion code from iOS as well (since we're modifying cross platform code here). There's not an easy way to do that at this point besides copying it unfortunately, as @cjhopman says :( I'd recommend just creating a new file in cxxreact and copying the function you need, then removing the dependency on `<jni/LocalString.h>`